generator/src/main.rs

```diff
@@ -878,6 +887,14 @@ impl Parser {
             }
         });
 
+        let consts_aliases = self.api_aliases.iter().map(|(name, value)| {
+            let alias_ident = Ident::new(&name[3..], Span::call_site());
+            let og_ident = Ident::new(&value[3..], Span::call_site());
+            quote! {
+                pub const #alias_ident: usize = #og_ident;
+            }
+        });
+
```

Comment:
AFAICT these aliases exist only for backwards compatibility. Because we can cleanly break compat, we shouldn't bother generating them.

Reply:
Should we break backward compatibility though? OpenXR itself tries to preserve it so maybe we should as well (at least for `sys`)?

Reply:
The OpenXR C headers operate under different constraints. We should focus on providing high quality support for the latest version.

Reply:
Okay, in FB's infinite wisdom, they didn't update the constants when used in their hand tracking capsule extension, so we're generating the original names:
```
pub struct HandCapsuleFB {
    pub points: [Vector3f; FB_HAND_TRACKING_CAPSULE_POINT_COUNT],
    pub radius: f32,
    pub joint: HandJointEXT,
}
```
and
```
pub struct HandTrackingCapsulesStateFB {
    pub ty: StructureType,
    pub next: *mut c_void,
    pub capsules: [HandCapsuleFB; FB_HAND_TRACKING_CAPSULE_COUNT],
}
```
(ass opposed to the "correct" `HAND_TRACKING_CAPSULE_POINT_COUNT_FB` and `HAND_TRACKING_CAPSULE_COUNT_FB`.

Do you want me to keep the alias constants generating in Rust or try to do a lookup when emitting these structures and try to fix it with the alias map?

Reply:
Oh, that's annoying, and should maybe be raised upstream. Ideally I would prefer to work around this in the generator rather than letting it influence our output.

Reply:
I asked FB if we could restructure the public code to use the new constant names, but that's probably a change that would take a month to get to public OpenXR-SDK.